scripts/scripts.js

```diff
@@ -496,15 +496,15 @@ export function buildFigure(blockEl) {
  * Loads JS and CSS for a block.
  * @param {Element} block The block element
  */
-export async function loadBlock(block, callback) {
+export async function loadBlock(block) {
   if (!block.getAttribute('data-block-loaded')) {
     block.setAttribute('data-block-loaded', true);
     const blockName = block.getAttribute('data-block-name');
     try {
       loadCSS(`/blocks/${blockName}/${blockName}.css`);
       const mod = await import(`/blocks/${blockName}/${blockName}.js`);
       if (mod.default) {
-        await mod.default(block, blockName, document, callback);
+        await mod.default(block, blockName, document);
```

Comment:
Just a thought: why we do not tell the block it is being loaded eager or lazy ? 

```
export async function loadBlock(block, eager=false) {
   ...
  await mod.default(block, blockName, document, eager);
}
```

If a block receives `eager=true`, it knows it will  "block" the whole page loading thus it might need to be built with more intelligence (like asking for images to be loaded eager)